engine/model.py

```diff
@@ -137,13 +137,20 @@ def _load_scripted_objects(self, tilemap: arcade.TileMap) -> None:
         self.scripted_objects = {}
         self.scene.add_sprite_list(SCRIPTED_OBJECTS, use_spatial_hash=True)
 
+        self.scene.add_sprite_list("Solid Objects", use_spatial_hash=True)
+
         sprites = []
+        solid_objects = []
 
         for obj in tilemap.object_lists.get(SCRIPTED_OBJECTS, []):
             if obj.name is None:
                 raise ValueError("Missing name attribute for scripted object.")
 
             sprite = self._create_object_sprite(obj, tilemap)
+
+            if (obj.properties is not None) and obj.properties.get("solid", False):
```

Comment:
Do the same `obj.properties` check that the next loop does, which is raising a `ValueError` if `properties` is not set.